app/MMB/src/components/HeaderShareButton.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import * as React from 'react';
+import { Share, Platform } from 'react-native';
+import {
+  TextIcon,
+  HeaderRightButton,
+  Color,
+  StyleSheet,
+} from '@kiwicom/mobile-shared';
+import { DateFormatter, GetDeviceLocale } from '@kiwicom/mobile-localization';
+
+import BookingDetailContext from '../context/BookingDetailContext';
+
+type PropsWithContext = {|
+  +arrivalCityId: string,
+  +arrivalTime: Date,
+|};
+
+class HeaderShareButton extends React.Component<PropsWithContext> {
+  onPress = () => {
+    const lang = GetDeviceLocale().split('-')[0] || 'en';
```

Comment:
The way how we work with device locale is starting to be really ugly... 😞

Reply:
You are right, I suggest, we add one method to the native code, to get just language, or native pass it as a prop, and we add it to context.